arm-batch/2017-01-01/swagger/BatchManagement.json

```diff
@@ -1150,10 +1151,15 @@
         "id": {
           "type": "string",
           "description": "The resource ID of the Azure key vault associated with the Batch account."
+        },
+        "url": {
+          "type": "string",
```

Comment:
@dsgouda Could you give me an example on a swagger spec which constraint on url/uri? I couldn't find it in this repo.